falcon/api.py

```diff
@@ -554,9 +556,23 @@ def handle(req, resp, ex, params):
                                      'method named "handle" that is a '
                                      'member of the given exception class.')
 
-        # Insert at the head of the list in case we get duplicate
-        # adds (will cause the most recently added one to win).
-        self._error_handlers.insert(0, (exception, handler))
+        try:
+            exception_tuple = tuple(exception)
+        except TypeError:
+            exception_tuple = tuple((exception, ))
```

Comment:
Sorry for being unclear here.

My suggestion was getting rid of this outer `tuple` altogether, i.e. `(exception,)` is already a `tuple`.
Let me illustrate with an example:
```python
>>> exception = ZeroDivisionError
>>> type((exception,))
<type 'tuple'>
>>> type(tuple((exception,)))
<type 'tuple'>
>>> (exception,) == tuple((exception,))
True
>>> (exception,) is tuple((exception,))
False
```

Reply:
yeah sure, sorry for not seeing that....! 🤦‍♂️